test/cmd.js

```diff
@@ -115,7 +115,7 @@ describe('express(1)', function () {
       it('should create basic app', function (done) {
         run(ctx.dir, [], function (err, output) {
           if (err) return done(err)
-          assert.equal(parseCreatedFiles(output, ctx.dir).length, 17)
```

Comment:
I wrote it that way for the sake of consistency. I don't think it removes the clarity of what's going - it's clear both ways - whether `bin/www` serves sort of as a `main()`(we know or what those does by convention), or you go to make changes to the server (as an abstract concept) at `server.js`. However, I think changing the file structure just when using es2015 syntax, and keeping it as is otherwise, could actually reduce clarity in a lot of cases. 

Additionally, I think it's not too idiosyncratic to have a single file, in this case `www`, that simply serves  as an entrypoint to all the things that need to be done at startup, especially if it's located underneath `bin/` (although I could be wrong, I don't have as much experience as you so please let me know. I've seen a lot of things like this, where command parsing and side effects are done in a single file that's the actual entry point but doesn't do anything else). Previously, only one thing had to be "done", i.e. start the server, but now two thing needs to be done, i.e. transpile the code, then start the server. In the future, perhaps more things will need to be done upon startup that don't involve starting the server or transpiling the code? FWIW, personally, if I'm trying to understand some new code, I usually look for such a file to follow the "signal flow", so to speak. 

So, intuitively, it felt like, given the circumstances of how `babel-core/register` works, no way around that aspect of the problem, that I would be adding complexity by doing the alternative, where as the way I have it here was consistent and still basically just as concise. Although obviously this up to your discretion, I'm happy to do whatever works best for you and the project.